signalfx-metrics/src/main/java/com/signalfx/metrics/SourceNameHelper.java

```diff
@@ -49,6 +52,9 @@ public static String getDefaultSourceName() {
 
     @Deprecated
     public static String getAwsInstanceId() throws IOException {
```

Comment:
Alternatively could just delete this method, it has been deprecated since 2016

Reply:
This is fine to me 🤷

Reply:
deleted it